packages/nx/src/command-line/examples.ts

```diff
@@ -284,6 +284,16 @@ export const examples: Record<string, Example[]> = {
       description:
         'Test all projects with tags starting with `api-`.  Note: your shell may require you to escape the `*` like this: `\\*`',
     },
+    {
+      command: 'nx run-many -t test --projects=tag:type:ui',
+      description:
+        'Test all projects with a `type:ui` tag'
+    },
+    {
+      command: 'nx run-many -t test --projects=tag:type:ui,tag:scope:shared',
+      description:
+        'Test all projects with `type:ui` and `scope:shared` tags',
```

Comment:
This would run for all projects with type:hi **_or_** scope:shared, the current wording makes it read like it would filter out projects which have type:ui, but not scope:shared

Reply:
Interesting, how would we make it run for AND?

Reply:
There's not support for requiring multiple tags.